test/integration/targets/dnf/tasks/dnfinstallroot.yml

```diff
@@ -1,7 +1,6 @@
 # make a installroot
 - name: Create installroot
-  local_action:
-    module: command mktemp -d "{{ lookup('env', 'TMPDIR') | default('/tmp', true) }}/ansible.test.XXXXXX"
+  command: mktemp -d "{{ lookup('env', 'TMPDIR') | default('/tmp', true) }}/ansible.test.XXXXXX"
```

Comment:
The lookup happens on the controller and not the remote host. This may become a problem in the future. Because we do fact gathering for Linux we may as well use the fact for TMP instead.

Reply:
Good catch. I was only checking for lookups using registered results from modules and completely overlooked the fact we were using a controller env var for the remote command. I'll fix these.